test/state/state.hpp

```diff
@@ -184,11 +200,11 @@ void finalize(State& state, evmc_revision rev, const address& coinbase,
 
 [[nodiscard]] std::variant<TransactionReceipt, std::error_code> transition(State& state,
     const BlockInfo& block, const Transaction& tx, evmc_revision rev, evmc::VM& vm,
-    int64_t block_gas_left);
+    int64_t block_gas_left, uint64_t blob_gas_left);
```

Comment:
Why did you make `blob_gas_left` unsigned, when `block_gas_left` is signed?

Reply:
fixed. No idea actually.